puppetlabs_spec_helper.gemspec

```diff
@@ -26,6 +26,7 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency 'pathspec', '>= 0.2.1', '< 1.1.0'
   spec.add_runtime_dependency 'puppet-lint', '~> 2.0'
   spec.add_runtime_dependency 'puppet-syntax', ['>= 2.0', '< 4']
+  spec.add_runtime_dependency 'rspec-github', '~> 2.0'
```

Comment:
As an alternative to making it a hard dependency, it can also be made a soft dependency:
```ruby
RSpec.configure do |c|
  if ENV['GITHUB_ACTIONS'] == 'true'
    begin
      c.formatter = 'RSpec::Github::Formatter'
    rescue LoadError
      # rspec-github not present
    end
  end
```

Reply:
Does it really matter if it is a hard dep?  As long as there is a flag to turn it off for other CI systems, I don't see the gem being pulled as much of a problem.

Reply:
If it isn't a hard dep, that would mean that `rspec-github` would have to be manually added to (literally) 100s of repos.

Reply:
> Does it really matter if it is a hard dep? As long as there is a flag to turn it off for other CI systems, I don't see the gem being pulled as much of a problem.

Every additional gem is a potential problem. Just wanted to highlight it.

> If it isn't a hard dep, that would mean that `rspec-github` would have to be manually added to (literally) 100s of repos.

That was also my consideration. This looked easier for the average case.